Examples/Scripts/Propagation/propagation_summary.py

```diff
@@ -0,0 +1,128 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import argparse
+import uproot
+import seaborn as sns
+
+p = argparse.ArgumentParser()
+
+p.add_argument(
+    "-i",
+    "--input",
+    nargs="+",
+    type=str,
+    default="",
+    help="Input file(s) with propagation summary",
+)
+
+p.add_argument(
+    "-l",
+    "--label",
+    nargs="+",
+    type=str,
+    default="",
+    help="Input label(s) for the propagation summary",
+)
+
+p.add_argument(
+    "-c",
+    "--color",
+    nargs="+",
+    type=str,
+    default="",
+    help="Input label(s) for the propagation summary",
+)
+
+p.add_argument(
+    "-m",
+    "--marker",
+    nargs="+",
+    type=str,
+    default=["o"],
+    choices=[
+        "o",
+        "s",
+        "D",
+        "v",
+        "^",
+        "<",
+        ">",
+        "p",
+        "P",
+        "*",
+        "h",
+        "H",
+        "+",
+        "x",
+        "X",
+        "d",
+    ],
+    help="Input label(s) for the propagation summary",
+)
+
+p.add_argument(
+    "-o",
+    "--output",
+    type=str,
+    default="",
+    help="Output file base name",
+)
+
+args = p.parse_args()
+
+try:
```

Comment:
Hmm this swallows any kind of error, also errors not related to files and labels...
cant we just `assert len(args.input) == len(args.label) == len(args.marker)` to ensure this?